spring-boot-project/spring-boot-docs/src/docs/asciidoc/actuator/metrics.adoc

```diff
@@ -611,165 +684,166 @@ You can also change the interval at which metrics are sent to Wavefront:
 
-TIP: In some cases, exceptions handled in Web controllers are not recorded as request metrics tags.
-Applications can opt-in and record exceptions by <<web#web.servlet.spring-mvc.error-handling, setting handled exceptions as request attributes>>.
+TIP: In some cases, exceptions handled in web controllers are not recorded as request metrics tags.
+Applications can opt in and record exceptions by <<web#web.servlet.spring-mvc.error-handling, setting handled exceptions as request attributes>>.
 
 
 
 [[actuator.metrics.supported.spring-webflux]]
 ==== Spring WebFlux Metrics
 Auto-configuration enables the instrumentation of all requests handled by Spring WebFlux controllers and functional handlers.
 By default, metrics are generated with the name, `http.server.requests`.
-The name can be customized by setting the configprop:management.metrics.web.server.request.metric-name[] property.
+You can customize the name by setting the configprop:management.metrics.web.server.request.metric-name[] property.
 
 `@Timed` annotations are supported on `@Controller` classes and `@RequestMapping` methods (see <<actuator#actuator.metrics.supported.timed-annotation>> for details).
-If you don't want to record metrics for all Spring WebFlux requests, you can set configprop:management.metrics.web.server.request.autotime.enabled[] to `false` and exclusively use `@Timed` annotations instead.
+If you do not want to record metrics for all Spring WebFlux requests, you can set configprop:management.metrics.web.server.request.autotime.enabled[] to `false` and exclusively use `@Timed` annotations instead.
 
 By default, WebFlux related metrics are tagged with the following information:
 
 |===
 | Tag | Description
 
 | `exception`
-| Simple class name of any exception that was thrown while handling the request.
+| The simple class name of any exception that was thrown while handling the request.
 
 | `method`
-| Request's method (for example, `GET` or `POST`)
+| The request's method (for example, `GET` or `POST`)
 
 | `outcome`
-| Request's outcome based on the status code of the response.
-  1xx is `INFORMATIONAL`, 2xx is `SUCCESS`, 3xx is `REDIRECTION`, 4xx `CLIENT_ERROR`, and 5xx is `SERVER_ERROR`
+| The request's outcome, based on the status code of the response.
+  1xx is `INFORMATIONAL`, 2xx is `SUCCESS`, 3xx is `REDIRECTION`, 4xx is `CLIENT_ERROR`, and 5xx is `SERVER_ERROR`
 
 | `status`
-| Response's HTTP status code (for example, `200` or `500`)
+| The response's HTTP status code (for example, `200` or `500`)
 
 | `uri`
-| Request's URI template prior to variable substitution, if possible (for example, `/api/person/\{id}`)
+| The request's URI template prior to variable substitution, if possible (for example, `/api/person/\{id}`)
 |===
 
-To add to the default tags, provide one or more ``@Bean``s that implement `WebFluxTagsContributor`.
+To add to the default tags, provide one or more `@Bean` instances that implement `WebFluxTagsContributor`.
```

Comment:
`@Bean` is an annotation so you can't really have instances of it. If you'd like to replace `@Bean`s, I think "beans" would be better. Either way, the next line should probably be changed too for consistency.